Gemfile.lock

```diff
@@ -73,6 +73,7 @@ GEM
     unicode-display_width (1.7.0)
 
 PLATFORMS
+  x86_64-darwin-19
```

Comment:
This is just a configuration that is auto-added by bundler. You have a new version of MacOS than I do.